src/player.js

```diff
@@ -0,0 +1,142 @@
+    let i = 0;
+    while (i < this.options.length) {
+      if (arr[0] === this.options[i][0] && arr[1] === this.options[i][1]) {
+        this.options.splice(i, 1);
+        i = 100;
+      }
+      i += 1;
+    }
+  },
+  makeMove(x, y, board, shipsArr) {
+    if (this.options.length > 0) {
+      this.removeFromOption([x, y]);
+      gameBoard.receiveAttack(x, y, board, shipsArr);
+    }
+  },
+  makeAttacks(ship, computer) {
+    const { board } = computer;
+    const elements = [];
+    if (ship.orientation === 'vertical') {
+      for (let i = ship.first[0]; i < ship.size + ship.first[0]; i += 1) {
+        elements.push([i, ship.first[1]]);
+      }
+    } else {
+      for (let i = ship.first[1]; i < ship.size + ship.first[1]; i += 1) {
+        elements.push([ship.first[0], i]);
+      }
+    }
+    elements.forEach((e) => {
+      if (!gameBoard.checkNull(e[0] + 1, e[1], board) && typeof board[e[0] + 1][e[1]] !== 'number') {
+        this.makeMove(e[0] + 1, e[1], board, ships);
+        document.getElementById(`C-${e[0] + 1}-${e[1]}`).className = 'water';
+      }
+      if (!gameBoard.checkNull(e[0] - 1, e[1], board) && typeof board[e[0] - 1][e[1]] !== 'number') {
+        this.makeMove(e[0] - 1, e[1], board, ships);
+        document.getElementById(`C-${e[0] - 1}-${e[1]}`).className = 'water';
+      }
+      if (!gameBoard.checkNull(e[0], e[1] + 1, board) && typeof board[e[0]][e[1] + 1] !== 'number') {
+        this.makeMove(e[0], e[1] + 1, board, ships);
+        document.getElementById(`C-${e[0]}-${e[1] + 1}`).className = 'water';
+      }
+      if (!gameBoard.checkNull(e[0], e[1] - 1, board) && typeof board[e[0]][e[1] - 1] !== 'number') {
+        this.makeMove(e[0], e[1] - 1, board, ships);
+        document.getElementById(`C-${e[0]}-${e[1] - 1}`).className = 'water';
+      }
+      if (!gameBoard.checkNull(e[0] + 1, e[1] + 1, board) && typeof board[e[0] + 1][e[1] + 1] !== 'number') {
+        this.makeMove(e[0] + 1, e[1] + 1, board, ships);
+        document.getElementById(`C-${e[0] + 1}-${e[1] + 1}`).className = 'water';
+      }
+      if (!gameBoard.checkNull(e[0] - 1, e[1] - 1, board) && typeof board[e[0] - 1][e[1] - 1] !== 'number') {
+        this.makeMove(e[0] - 1, e[1] - 1, board, ships);
+        document.getElementById(`C-${e[0] - 1}-${e[1] - 1}`).className = 'water';
+      }
+      if (!gameBoard.checkNull(e[0] + 1, e[1] - 1, board) && typeof board[e[0] + 1][e[1] - 1] !== 'number') {
+        this.makeMove(e[0] + 1, e[1] - 1, board, ships);
+        document.getElementById(`C-${e[0] + 1}-${e[1] - 1}`).className = 'water';
+      }
+      if (!gameBoard.checkNull(e[0] - 1, e[1] + 1, board) && typeof board[e[0] - 1][e[1] + 1] !== 'number') {
+        this.makeMove(e[0] - 1, e[1] + 1, board, ships);
+        document.getElementById(`C-${e[0] - 1}-${e[1] + 1}`).className = 'water';
+      }
+    });
+  },
+  gameOver() {
+    return gameBoard.allShipsSunk(ships);
+  },
+});
+
+module.exports = Player;
```

Comment:
We need to use ES6, export, import, syntax, as stated in the [Pathwright to this project](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622118/)
![image](https://user-images.githubusercontent.com/46115725/73208810-b2005280-410c-11ea-8801-b9168287bdcf.png)